src/modules/sign-in/SignIn.jsx

```diff
@@ -63,10 +77,50 @@ export class SignIn extends Component {
         </Grid>
       )
     }
+  };
+
+  onChange = (e) => {
+   const name = e.target.name;
+   const value = e.target.value;
```

Comment:
useless consts just set state directly from target

Reply:
done
onChange = (e) => {
   this.setState({[e.target.name]: e.target.value});
  }